src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -0,0 +1,1560 @@
+        std::vector<std::string> graphFiles = utils.getListOfFilesInDirectory(aggregatorFilePath);
+
+        std::vector<std::string>::iterator graphFilesIterator;
+        std::string compositeFileNameFormat = graphId + "_compositecentralstore_";
+
+        for (graphFilesIterator = graphFiles.begin(); graphFilesIterator != graphFiles.end(); ++graphFilesIterator) {
+            std::string graphFileName = *graphFilesIterator;
+
+            if ((graphFileName.find(compositeFileNameFormat) == 0) && (graphFileName.find(".gz") != std::string::npos)) {
+                compositeCentralStoreFiles.push_back(graphFileName);
+            }
+        }
+        fileCombinations = getCombinations(compositeCentralStoreFiles);
+    }
+
+    std::map<string, std::vector<string>> partitionMap;
+
+    for (std::vector<vector<pair<string, string>>>::iterator i = results.begin(); i != results.end(); ++i) {
+        std::vector<pair<string, string>> rowData = *i;
+        string host = "";
+
+        string workerID = rowData.at(0).second;
+        string name = rowData.at(1).second;
+        string ip = rowData.at(2).second;
+        string user = rowData.at(3).second;
+        string serverPort = rowData.at(4).second;
+        string serverDataPort = rowData.at(5).second;
+        string partitionId = rowData.at(6).second;
+
+        if ((ip.find("localhost") != std::string::npos) || ip == masterIP) {
+            host = ip;
+        } else {
+            host = user + "@" + ip;;
+        }
+
+        if( partitionMap.find(workerID) == partitionMap.end()){
+            std::vector<string> partitionVec;
+            partitionVec.push_back(partitionId);
+            partitionMap.insert(std::pair<string, std::vector<string>>(workerID, partitionVec));
+        } else {
+            std::vector<string> partitionVec = partitionMap.find(workerID)->second;
+            partitionVec.push_back(partitionId);
+        }
+
+        triangleCount_logger.log("###TRIANGLE-COUNT-EXECUTOR### Getting Triangle Count : Host " + host + " Server Port " +
+                            serverPort + " PartitionId " + partitionId, "info");
+
+    }
+
+    for (auto &&futureCall:remoteCopyRes) {
+        futureCall.wait();
+    }
+
+    for (int i = 0; i < workerListSize; i++) {
+        Utils::worker currentWorker = workerList.at(i);
+        string host = currentWorker.hostname;
+        string workerID = currentWorker.workerID;
+        string partitionId;
+
+        std::vector<string> partitionList = partitionMap[workerID];
+
+        std::vector<string>::iterator partitionIterator;
+
+        for (partitionIterator = partitionList.begin(); partitionIterator != partitionList.end(); ++partitionIterator) {
+            partitionCount++;
+            int workerPort = atoi(string(currentWorker.port).c_str());
+            int workerDataPort = atoi(string(currentWorker.dataPort).c_str());
+            triangleCount_logger.log("====>PORT:" + std::to_string(workerPort), "info");
```

Comment:
This message does not have any meaning. If its required please rephrase. Else lets remove it.

Reply:
Removed the log from that place.